src/main/kotlin/app/revanced/cli/command/PatchCommand.kt

```diff
@@ -316,6 +316,7 @@ internal object PatchCommand : Runnable {
         // region Save
         apk.copyTo(temporaryFilesPath.resolve(apk.name), overwrite = true).apply {
             patcherResult.applyTo(this)
+            patcherResult.dexFiles.forEach{it.stream.close()}   
```

Comment:
Maybe it.use {} should be used in the applyTo function in ReVanced Library instead so it auto closes after using the stream, but I am not sure if closing the streams is applyTo's job. I think a better approach would be to return files instead of opening streams to them so that one can open streams for it at any time